CiJenkinsfile

```diff
@@ -0,0 +1,193 @@
+import java.text.SimpleDateFormat
```

Comment:
This repo will be used only for that chapter so you can change the file to Jenkinsfile.